packages/server/tests/acceptance/rpc_batch3.spec.ts

```diff
@@ -1056,4 +1073,809 @@ describe('@api-batch-3 RPC Server Acceptance Tests', function () {
+        });
+
+        it('should be able to debug a failing CREATE transaction of type Legacy with call depth and onlyTopCall true', async function () {
+          const transaction = {
+            ...transactionTypeLegacy,
+            nonce: await relay.getAccountNonce(accounts[0].address, requestId),
+            chainId: 0x12a,
+            from: accounts[0].address,
+            gasPrice: await relay.gasPrice(requestId),
+            data: '0x01121212',
+          };
+
+          let signedTransaction = await accounts[0].wallet.signTransaction(transaction);
+          let transactionHash = await relay.sendRawTransaction(signedTransaction, requestId);
+
+          const resultDebug = await relay.call(
+            RelayCalls.ETH_ENDPOINTS.DEBUG_TRACE_TRANSACTION,
+            [transactionHash, { tracer: callTracer, tracerConfig: tracerConfigTrue }],
+            requestId,
+          );
+
+          failingResultCreate.from = accounts[0].address;
+          failingResultCreate.input = '0x01121212';
+
+          Assertions.validateResultDebugValues(resultDebug, ['to', 'output'], [], failingResultCreate);
+        });
+
+        it('should be able to debug a failing CALL transaction of type Legacy with call depth and onlyTopCall true', async function () {
+          const transaction = {
+            ...transactionTypeLegacy,
+            from: accounts[0].address,
+            to: reverterEvmAddress,
+            nonce: await relay.getAccountNonce(accounts[0].address, requestId),
+            gasPrice: await relay.gasPrice(requestId),
+            data: '0x0323d234',
+          };
+
+          let signedTransaction = await accounts[0].wallet.signTransaction(transaction);
+          let transactionHash = await relay.sendRawTransaction(signedTransaction, requestId);
+
+          const resultDebug = await relay.call(
+            RelayCalls.ETH_ENDPOINTS.DEBUG_TRACE_TRANSACTION,
+            [transactionHash, { tracer: callTracer, tracerConfig: tracerConfigTrue }],
+            requestId,
+          );
+
+          failingResultCall.from = accounts[0].address;
+          failingResultCall.input = '0x0323d234';
+
+          Assertions.validateResultDebugValues(resultDebug, ['to', 'output', 'calls'], [], failingResultCall);
+        });
+      });
+
+      describe('Test transaction of type 1', async function () {
+        //onlyTopCall:false
+        it('should be able to debug a successful CREATE transaction of type 2930 with call depth and onlyTopCall false', async function () {
+          const transaction = {
+            ...transactionType2930,
+            chainId: 0x12a,
+            data: bytecode,
+            nonce: await relay.getAccountNonce(accounts[0].address, requestId),
+            gasPrice: await relay.gasPrice(requestId),
+          };
+
+          let signedTransaction = await accounts[0].wallet.signTransaction(transaction);
```

Comment:
@anastasiya-kovaliova You can use const for the variables that do not get reassigned, like thos two : )) There were other places as well